layer/contracts/models.py

```diff
@@ -225,11 +133,6 @@ def with_project_name(self, project_name: str) -> "Model":
         new_model._update_with(new_asset)
         return new_model
 
-    def with_language_version(self, language_version: Tuple[int, int, int]) -> "Model":
```

Comment:
you can delete this from datasets.py too

Reply:
saving this for future cleanup